application/src/main/java/org/thingsboard/server/service/entitiy/DefaultTbNotificationEntityService.java

```diff
@@ -235,11 +252,6 @@ public void notifyCreateOrUpdateOrDeleteRelation(TenantId tenantId, CustomerId c
         }
     }
 
-    private <E extends HasName, I extends EntityId> void logEntityAction(TenantId tenantId, I entityId, E entity, CustomerId customerId,
```

Comment:
why you removed this?

Reply:
https://github.com/thingsboard/thingsboard/pull/6602#discussion_r884519621

Reply:
Ok